src/components/VLogoLoader/VLogoLoader.vue

```diff
@@ -39,16 +39,24 @@
   </svg>
 </template>
 
-<script>
-import { defineComponent } from '@nuxtjs/composition-api'
+<script lang="ts">
+import { defineComponent, PropType } from '@nuxtjs/composition-api'
 
 import { useReducedMotion } from '~/composables/use-media-query'
 
-import { propTypes } from './VLogoLoader.types'
-
 export default defineComponent({
   name: 'VLogoLoader',
-  props: propTypes,
+  props: {
+    status: {
+      type: String as PropType<'loading' | 'idle'>,
+      default: 'idle',
+      required: false,
```

Comment:
`required: false` is the [default behavior](https://vuejs.org/guide/components/props.html#prop-validation).